[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.jM8q2B9f1J"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(194): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/wpblast/plugin.php(196): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#7 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#8 {main}</pre>\n for query ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash) made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(199): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(204): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(209): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(194): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/wpblast/plugin.php(196): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#7 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#8 {main}</pre>\n for query ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash) made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(199): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(204): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(209): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:36 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): WPTweetImporter::activation(false)\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wptwee...', false)\n#4 /internal/eval.php(20): activate_plugin('wptweetimporter...')\n#5 {main}\n  thrown in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php on line 34\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"wptweetimporter\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.vGOuP36WAC"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(194): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/wpblast/plugin.php(196): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#7 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#8 {main}</pre>\n for query ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash) made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(199): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(204): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(209): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(194): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/wpblast/plugin.php(196): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#7 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#8 {main}</pre>\n for query ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash) made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(199): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(204): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(209): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:36 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): WPTweetImporter::activation(false)\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wptwee...', false)\n#4 /internal/eval.php(20): activate_plugin('wptweetimporter...')\n#5 {main}\n  thrown in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php on line 34",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:43 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): WPTweetImporter::activation(false)\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wptwee...', false)\n#4 /internal/eval.php(20): activate_plugin('wptweetimporter...')\n#5 {main}\n  thrown in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php on line 34\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"wptweetimporter\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.MhJH4I376R"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(194): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/wpblast/plugin.php(196): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#7 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#8 {main}</pre>\n for query ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash) made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(199): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(204): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:08 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(209): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(194): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Hash' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash)</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Data type expected in MySQL query, unknown token received: WPBlast_Sitemap_Hash.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/wpblast/plugin.php(196): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#3 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#7 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#8 {main}</pre>\n for query ALTER TABLE wp_wpblast_sitemap ADD CONSTRAINT WPBlast_Sitemap_Hash UNIQUE (hash) made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(199): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_CacheExpiration' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(204): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Url' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Unknown show type: INDEXES FROM.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:21 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active'</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: INDEXES FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3065): WP_SQLite_DB->query('SHOW INDEXES FR...')\n#2 /wordpress/wp-content/plugins/wpblast/plugin.php(209): wpdb->get_row('SHOW INDEXES FR...', 'ARRAY_A')\n#3 /wordpress/wp-content/plugins/wpblast/plugin.php(70): wpblast_install()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): wpblast_register_activation(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wpblas...', false)\n#8 /internal/eval.php(20): activate_plugin('wpblast/plugin....')\n#9 {main}</pre>\n for query SHOW INDEXES FROM wp_wpblast_sitemap WHERE key_name = 'WPBlast_Sitemap_Active' made by activate_plugin, do_action('activate_wpblast/plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wpblast_register_activation, wpblast_install, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:36 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): WPTweetImporter::activation(false)\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wptwee...', false)\n#4 /internal/eval.php(20): activate_plugin('wptweetimporter...')\n#5 {main}\n  thrown in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php on line 34",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:43 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): WPTweetImporter::activation(false)\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wptwee...', false)\n#4 /internal/eval.php(20): activate_plugin('wptweetimporter...')\n#5 {main}\n  thrown in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php on line 34",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "wptweetimporter",
    "details": "[25-May-2025 04:35:50 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function wp_insert_category() in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php:34\nStack trace:\n#0 /wordpress/wp-includes/class-wp-hook.php(324): WPTweetImporter::activation(false)\n#1 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#2 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#3 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_wptwee...', false)\n#4 /internal/eval.php(20): activate_plugin('wptweetimporter...')\n#5 {main}\n  thrown in /wordpress/wp-content/plugins/wptweetimporter/includes/main.php on line 34\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"wptweetimporter\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/w/wptweetimporter/error.json"
  }
]
